### PR TITLE
docs(puppeteer-web): a note about running inside extension

### LINKED ISSUE #3455: No errors are thrown on asyncawait check.
**Origin of problem:** `puppeteer-web` on chrome extension.
**Source:** https://github.com/GoogleChrome/puppeteer/blob/master/index.js#L21
**Relevent Stackoverflow Question:** https://stackoverflow.com/q/53001009/6161265

The bundled version shows anonymous function, figuring out the problem wasn't smooth.

image

**Possible solutions:**
- It should notify the user somehow. Either by an argument or so in the browser.
- Let the user know about possible gimmicks of trying this inside chrome extension, which is to let `unsafe-eval` on content_security_policy of their `manifest.json` file.

```js
"content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'"
```

### PR DESCRIPTION
**Pull request by aslushnikov**
Fixes #3455

### CODE REVIEW

File: utils/browser/README.md [hunk 1]
```diff
@@ -24,3 +24,13 @@ another browser instance through its WS Endpoint:
 See our [puppeteer-web tests](https://github.com/GoogleChrome/puppeteer/blob/master/utils/browser/test.js)
 for details.
 
+### Running inside Chrome Extension
+
+You might want to enable `unsafe-eval` inside extension by adding the following
+to your`manifest.json` file:
```
Comment (vsemozhetbyt) [inline utils/browser/README.md]:
```suggestion
to your `manifest.json` file:
```

File: utils/browser/README.md [hunk 2]
```diff
@@ -24,3 +24,13 @@ another browser instance through its WS Endpoint:
 See our [puppeteer-web tests](https://github.com/GoogleChrome/puppeteer/blob/master/utils/browser/test.js)
 for details.
 
+### Running inside Chrome Extension
+
+You might want to enable `unsafe-eval` inside extension by adding the following
```
Comment (JoelEinbinder) [inline utils/browser/README.md]: the extension

**Review by JoelEinbinder — APPROVED** [review]

File: utils/browser/README.md (hunk 1 above)
Comment (aslushnikov) [reply utils/browser/README.md]: Done, thanks!

File: utils/browser/README.md (hunk 2 above)
Comment (aslushnikov) [reply utils/browser/README.md]: Done

**Pull request merged**